executor/builder.go

```diff
@@ -2164,7 +2164,12 @@ func (b *executorBuilder) buildAnalyzeIndexIncremental(task plannercore.AnalyzeI
 }
 
 func (b *executorBuilder) buildAnalyzeSamplingPushdown(task plannercore.AnalyzeColumnsTask, opts map[ast.AnalyzeOptionType]uint64, autoAnalyze string, schemaForVirtualColEval *expression.Schema) *analyzeTask {
-	job := &statistics.AnalyzeJob{DBName: task.DBName, TableName: task.TableName, PartitionName: task.PartitionName, JobInfo: autoAnalyze + "analyze table"}
+	colStrs := make([]string, 0, len(task.ColsInfo))
+	for _, col := range task.ColsInfo {
+		colStrs = append(colStrs, col.Name.L)
+	}
+	columnsInfo := "columns " + strings.Join(colStrs, ",")
+	job := &statistics.AnalyzeJob{DBName: task.DBName, TableName: task.TableName, PartitionName: task.PartitionName, JobInfo: autoAnalyze + "analyze table " + columnsInfo + " with " + analyzeOptionsToString(opts)}
```

Comment:
We may not test this part yet? The result of `SHOW ANALYZE STATUS`

Reply:
It's used in analyze asserts.